### ISSUE TITLE
Cannot create a node with latest version of Jenkins and Credentials plugin

### ISSUE THREAD

**Issue by kshtsk** [pycontribs/jenkinsapi#824]
# Bug Report #
##### Jenkins 2.361

Cannot create a node with latest version of Jenkins and Credentials plugin.

```
Failed request at http://X.X.X.X:8080/credential-store/domain/_/api/python?depth=2 with params: None
Traceback (most recent call last):
  File "/home/YYYYY/v/bin/jcs", line 8, in <module>
    sys.exit(main())
  File "/home/YYYYY/v/lib/python3.10/site-packages/jcs/__init__.py", line 289, in main
    args.func(args)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jcs/__init__.py", line 192, in _do_create
    jen_client.create_node(
  File "/home/YYYYY/v/lib/python3.10/site-packages/jcs/jen.py", line 70, in create_node
    node = self._client.nodes.create_node(node_name, node_dict)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/nodes.py", line 152, in create_node
    urlencode(node.get_node_attributes())))
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/node.py", line 118, in get_node_attributes
    credential = self.jenkins.credentials[
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/jenkins.py", line 616, in credentials
    return self.get_credentials(Credentials2x)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/jenkins.py", line 609, in get_credentials
    return Credentials(url, self)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/credentials.py", line 30, in __init__
    JenkinsBase.__init__(self, baseurl)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/jenkinsbase.py", line 38, in __init__
    self.poll()
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/jenkinsbase.py", line 60, in poll
    data = self._poll(tree=tree)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/credentials.py", line 36, in _poll
    data = self.get_data(url, tree=tree)
  File "/home/YYYYY/v/lib/python3.10/site-packages/jenkinsapi/jenkinsbase.py", line 84, in get_data
    response.raise_for_status()
  File "/home/YYYYY/v/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http://X.X.X.X:8080/credential-store/domain/_/api/python?depth=2
```

**Comment by kshtsk** [comment]
Fix is in the PR https://github.com/pycontribs/jenkinsapi/pull/823

**Comment by lechat** [comment]
This is fixed in release 0.3.12
